etp-backend/src/main/clj/solita/etp/service/pregenerated_csv.clj

```diff
@@ -0,0 +1,54 @@
+(ns solita.etp.service.pregenerated-csv
+  (:require [clojure.java.io :as io]
+            [clojure.tools.logging :as log]
+            [solita.etp.service.file :as file-service]
+            [solita.etp.service.energiatodistus-csv :as energiatodistus-csv-service]
+            [solita.etp.service.kayttaja :as kayttaja-service])
+  (:import (java.io File)))
+
+(def pregenerator-whoami {:id (kayttaja-service/system-kayttaja :pregenerator)
+                          :rooli -1})
+
+(def packages
+  [{:destination "energiatodistukset-ids.csv" ;; TODO Remove, this is only for convenience of development
```

Comment:
Näitä polkuja voisi vielä tarkastella, varmaan ei kannattaisi laittaa ihan juureen.